src/main/java/org/isf/patient/mapper/PatientMapper.java

```diff
@@ -70,6 +71,7 @@ public Patient map2Model(PatientDTO toObj) {
 			photo.setPhoto(toObj.getBlobPhoto());
 			patient.setPatientProfilePhoto(photo);
 		}
+		patient.setPatientConsensus(new PatientConsensus(toObj.isConsensusFlag(), toObj.isConsensusAdministrativeFlag(), toObj.isConsensusServiceFlag(), patient));
```

Comment:
Until we don't have the "ui" ready for sending the patient consensus flags, we should force to true the first one in the PatientController